tests/foreman/ui/test_settings.py

```diff
@@ -98,7 +98,6 @@ def test_positive_update_restrict_composite_view(session, setting_update, repo_s
                     session.contentview.delete(content_view_name)
 
 
-@pytest.mark.skip_if_open("BZ:1677282")
```

Comment:
Shouldn't we keep the BZ numbers at least in docstrings (:BZ:)?

Reply:
ah ok, it is there, on line 112